test/evp_kdf_test.c

```diff
@@ -1603,6 +1604,9 @@ static int test_kdf_ss_kmac(void)
 
     *p++ = OSSL_PARAM_construct_utf8_string(OSSL_KDF_PARAM_MAC,
                                             (char *)OSSL_MAC_NAME_KMAC128, 0);
+    /* The digest parameter is not needed here and should be ignored */
+    *p++ = OSSL_PARAM_construct_utf8_string(OSSL_KDF_PARAM_DIGEST,
+                                            (char *)"SHA256", 0);
```

Comment:
Maybe use some bogus digest name?

Reply:
What I was trying to do here is just test that it doesnt fail and it gets the correct output. I dont think it should care what it does with the digest (whether the implementation 'completely' ignores it or not does not really matter).

Reply:
OK